relayer/chains/cosmos/msg_handlers_connection.go

```diff
@@ -0,0 +1,83 @@
+package cosmos
+
+import (
+	conntypes "github.com/cosmos/ibc-go/v3/modules/core/03-connection/types"
+	"github.com/cosmos/relayer/v2/relayer/processor"
+	"github.com/cosmos/relayer/v2/relayer/provider/cosmos"
+	"go.uber.org/zap"
+)
+
+func (ccp *CosmosChainProcessor) handleMsgConnectionOpenInit(p msgHandlerParams) bool {
+	ci := p.messageInfo.(*connectionInfo)
+	k := ci.connectionKey()
+	// Construct the start of the MsgConnectionOpenTry for the counterparty chain.
+	// PathProcessor will determine if this is needed.
+	// For example, if a MsgConnectionOpenTry or MsgConnectionOpenConfirm is not detected on the counterparty chain, and
+	// a MsgConnectionOpenAck is not detected yet on this chain,
+	// a MsgConnectionOpenTry will be sent to the counterparty chain
+	// using this information with the connection init proof from this chain added.
```

Comment:
These comments almost read like they would be better off as Go doc top-level comments for the entire methods.

Reply:
good call, will update